<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu_cliente</class>
 <widget class="QWidget" name="Menu_cliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>443</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Menu de Cliente</string>
  </property>
  <property name="windowOpacity">
   <double>0.000000000000000</double>
  </property>
  <widget class="QPushButton" name="boton_alta_cliente">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>721</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>721</width>
     <height>51</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>721</width>
     <height>51</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Alta de Cliente</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_modifica_cliente">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>721</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>721</width>
     <height>51</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>721</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Modificacion de Cliente</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_baja_cliente">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>721</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Baja de Cliente</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>boton_alta_cliente</sender>
   <signal>clicked()</signal>
   <receiver>Menu_cliente</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_modifica_cliente</sender>
   <signal>clicked()</signal>
   <receiver>Menu_cliente</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>800</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
